examples/src/main/scala/org/apache/spark/examples/snappydata/CreateColumnTable.scala

```diff
@@ -0,0 +1,219 @@
+    // "PARTITION_BY" attribute specifies partitioning key for CUSTOMER table(C_CUSTKEY),
+    // "BUCKETS" attribute specifies the smallest unit that
+    // can be moved around in SnappyStore when the data migrates. Here we specify
+    // the table to have 11 buckets
+    // For complete list of table attributes refer the documentation
+    snc.sql("CREATE TABLE CUSTOMER ( " +
+        "C_CUSTKEY     INTEGER NOT NULL," +
+        "C_NAME        VARCHAR(25) NOT NULL," +
+        "C_ADDRESS     VARCHAR(40) NOT NULL," +
+        "C_NATIONKEY   INTEGER NOT NULL," +
+        "C_PHONE       VARCHAR(15) NOT NULL," +
+        "C_ACCTBAL     DECIMAL(15,2)   NOT NULL," +
+        "C_MKTSEGMENT  VARCHAR(10) NOT NULL," +
+        "C_COMMENT     VARCHAR(117) NOT NULL)" +
+        "USING COLUMN OPTIONS (PARTITION_BY 'C_CUSTKEY', BUCKETS '11' )")
+
+    // insert some data in it
+    pw.println()
+    pw.println("Loading data in CUSTOMER table from a text file with delimited columns")
+    val tableSchema = snc.table("CUSTOMER").schema
+    val customerDF = snc.read.
+        format("com.databricks.spark.csv").schema(schema = tableSchema).
+        load(s"quickstart/src/resources/customer.csv")
+    customerDF.write.insertInto("CUSTOMER")
+
+    pw.println()
+    var result = snc.sql("SELECT COUNT(*) FROM CUSTOMER").collect()
+    pw.println("Number of records in CUSTOMER table after loading data are " + result(0).get(0))
+
+    pw.println()
+    pw.println("Inserting a row using INSERT SQL")
+    snc.sql("INSERT INTO CUSTOMER VALUES(20000, 'Customer20000', " +
+        "'Chicago, IL', 1, '555-101-782', 3500, 'MKTSEGMENT', '')")
+
+    pw.println()
+    result = snc.sql("SELECT COUNT(*) FROM CUSTOMER").collect()
+    pw.println("Number of records in CUSTOMER table are " + result(0).get(0))
+
+    pw.println("****Done****")
+  }
+
+  /**
+   * Creates a column table where schema is inferred from parquet data file
+   */
+  def createColumnTableInferredSchema(snc: SnappyContext, pw: PrintWriter): Unit = {
+    pw.println()
+
+    pw.println("****Create a column table using API where schema is inferred from parquet file****")
+    // create a partitioned row table using SQL
+    pw.println()
+    pw.println("Creating a column table(CUSTOMER) using API ")
+    snc.dropTable("CUSTOMER", ifExists = true)
+
+    val customerDF = snc.read.parquet(s"quickstart/src/resources/customer_parquet")
+
+    // props1 map specifies the properties for the table to be created
+    // "PARTITION_BY" attribute specifies partitioning key for CUSTOMER table(C_CUSTKEY),
+    // "BUCKETS" attribute specifies the smallest unit that can be moved around in
+    // SnappyStore when the data migrates. Here we configure the table to have 11 buckets
+    // For complete list of attributes refer the documentation
+    val props1 = Map("PARTITION_BY" -> "C_CUSTKEY", "BUCKETS" -> "11")
+    customerDF.write.format("column").mode("append").options(props1).saveAsTable("CUSTOMER")
```

Comment:
This implicit creation via saveAsTable should be avoided everywhere in our examples. Change above to: customerDF.write.insertInto("CUSTOMER") and create via "snc.createTable or snc.sql". If the only point of this example is showing implicit creation via saveAsTable, I think this should be avoided and example removed because implicit creation is a route we don't want users to adopt.